packages/analytics-js/src/services/ErrorHandler/processError.ts

```diff
@@ -46,6 +29,30 @@ const processError = (error: SDKError): string => {
   return errorMessage;
 };
 
+const normalizeErrorMessageForUnhandledError = (error: SDKError): string => {
+  let errorMessage;
```

Comment:
Add exception handling in this method similar to the `processError`.

Reply:
Addressed